### ISSUE TITLE
Send email on booking completion

### ISSUE THREAD

**Issue by utkarsh867** [rcltech/owl#61]

**Comment by welvin21** [comment]
I realised that this functionality should be done on phoenix side, hence I pushed a commit to phoenix. What we can do from owl is sending a post request to `'/email'` (I'm not so sure if it can be done using graphql mutation instead). 

Things to note when testing the functionality : 

1. Add `EMAIL_PASSWORD=<our owl.rctech.club yandex password>` to .env file.
2. Data model which is passed to the function is as follows : 
`const bookingData = {
    recipientDetails: {
      email: "test@email.com",
      firstName: "",
      lastName: ""
    },
    bookingDetails: {
      roomDetails: {
        roomName: "",
        roomNumber: ""
      },
      start: new Date(),
      end: new Date()
    }
  }
`
3. Here's what I got : 
![image](https://user-images.githubusercontent.com/47080921/72061197-5cf0ce00-3310-11ea-9717-d7407b6e081f.png)
